### ISSUE TITLE
[Feature Request] List of files with versions

### ISSUE THREAD

**Issue by xthrk** [FilenCloudDienste/filen-drive#11]
It would be nice to get an overview / a list of all files that have versions. And the possibility in this list to delete versions per file quickly (maybe via a tree view and delete the version fast and direct with an delete-button). There is a function in the settings to delete all versions (Settings, Versioned Files, Delete) so that all old versions are deleted and only the newest version is left, but I cannot see which files have versions that will be all deleted. I would like to be able to see directly the files that have versions, so that I can search for files that have old versions and delete those old versions only on specific files.

**Comment by Dwynr** [comment]
Yes, we are looking into different ways to improve version management.
